components/assistant-ui/markdown-text.tsx

```
@@ -91,12 +141,12 @@ export const MarkdownText: FC<{ text: string }> = () => {
         ),
         // Inline code (not code blocks - those use SyntaxHighlighter)
         code: ({ children, ...props }) => (
-          <code
-            className="rounded bg-terminal-dark/10 px-1.5 py-0.5 text-sm font-mono text-terminal-dark"
-            {...props}
-          >
-            {children}
-          </code>
+          <SmartInlineCode
+            children={children}
```

Biome (2.4.4)

[error] Avoid passing children using a prop

(lint/correctness/noChildrenProp)